### PR TITLE
Show Property manager names and tenant count on properties grid

### LINKED ISSUE #670: Properties table is not showing "Property Managers" or "Tenants"
**Describe the bug**

**To Reproduce**
1. Go to the Properties page
2. See missing column data

**Expected behavior**
We want to map the appropriate data to the columns and match it to the mockups here:
image

**To do**
Implement the expected behavior
Write system test that creates a property and then views the index page.

### PR DESCRIPTION
**Pull request by Chris-Boe**
### What issue is this solving?
Please connect this Pull Request to the issue it is resolving by using the controls on the right side of this PR.
Fixes #670 

### Any helpful knowledge/context for the reviewer?
Is a `npm install` necessary? n
Any special requirements to test? n

### Please make sure you've attempted to meet the following coding standards
- [x] Code builds successfully
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code
- [x] There aren't any unnecessary commits or files changed (shouldn't touch the /stashed folder unless the issue requests it)

If you're having trouble meeting this criteria, feel free to reach out on Slack for help!

### CODE REVIEW

**Comment by scybert** [comment]
tsx?

File: src/views/properties/properties.jsx [hunk 1]
```diff
@@ -56,13 +56,19 @@ const makeAuthHeaders = ({ user }) => ({ headers: { 'Authorization': `Bearer ${u
 
 const getDisplayProperties = (properties, showArchived) => properties.filter(p => showArchived || !p.archived);
 
+const formatPropertyManagerNames = (propertyManagers) =>
```
Comment (scybert) [inline src/views/properties/properties.jsx:59]: Why parens? @Chris-Boe

**Review by scybert — CHANGES_REQUESTED** [review]

File: src/views/properties/properties.jsx [hunk 2]
```diff
@@ -56,13 +56,19 @@ const makeAuthHeaders = ({ user }) => ({ headers: { 'Authorization': `Bearer ${u
...
+  propertyManagers.map(pm => `${pm.firstName} ${pm.lastName}`).join(', ');
+
+const formatTenantCount = (leases) =>
+  leases.map(l => l.occupants).reduce((sum, occupants) => sum + occupants);
```
Comment (scybert) [inline src/views/properties/properties.jsx:63]: Right here you use no parens for `l` and then before parens for `leases`. Pick one!

**Comment by Chris-Boe** [comment]
> tsx?

Unfortunately, we don't use typescript in this application

File: src/views/properties/properties.jsx (hunk 1 above)
Comment (Chris-Boe) [reply src/views/properties/properties.jsx:59]: This is pretty standard form and a personal preference as well. When we have a function with no parameters we use empty parentheses, when we have more than one parameter we use parentheses. Removing the parentheses only in the case where you have one parameter is somewhat an antipattern, especially for newcomers to coding

**Review by scybert — APPROVED** [review]

**Pull request merged**